packages/unigraph-dev-explorer/src/components/Request.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button } from '@material-ui/core';
+import { Code } from '@material-ui/icons';
+import React from 'react'
+import CodeMirror from 'react-codemirror'
+
+export default function Request () {
+    let [code, setCode]: [string, Function] = React.useState("// Your request here...")
+    
+    function onCodeChanged (newValue: string, change: any)  {
+        setCode(newValue)
+    }
```

Comment:
not a big deal so early on, but generally should use `useCallback` instead of defining functions directly inside of a component – it prevents unnecessary rerenders, which can slow down the applications once it gets large enough

Reply:
Thanks for pointing it out! I changed it to inline function in the component (since the operation is pretty short anyway), but we can definitely use more of standard React practices from now on!